server.js

```diff
@@ -2,11 +2,25 @@ const express = require('express')
 const bodyParser = require('body-parser')
 const path = require('path')
 const { reset } = require('nodemon');
+const mongoose = require('mongoose');
+const{ Schema } = mongoose;
+
+mongoose.connect('')
+
+//model for 'LoginInfo' table
+var userModel = mongoose.model('LoginInfo', new Schema({
+   userName: String,
+   password: String,
+   email: String,
+   authenticated: Number,
+   authenticateTime: String
+}), 'LoginInfo');
 
 let app = express()
 
 app.use(express.static(path.join(__dirname, "./")))
-app.use(bodyParser.text())
+app.use(bodyParser.urlencoded({ extended: false }))
+//this won't work for multiple users, once one user is authenticated it will read true for all users
```

Comment:
Have you looked up any videos or guides on making multi-user authentication?

Reply:
going to do that with the database, I wrote a function for it